skeleton/backend/backend.cabal

```diff
@@ -1,9 +1,21 @@
+cabal-version: 2.2
```

Comment:
https://cabal.readthedocs.io/en/3.4/cabal-package.html#common-stanzas

Given the shell provides cabal 3.4, which is some 3 years old now, I think we could safely bump to version 3.0 to allow wildcards (e.g. exposed-modules)